core/src/main/java/org/frankframework/jdbc/XmlQuerySender.java

```diff
@@ -73,6 +74,7 @@
  * }</pre>
  * <br/>
  *
+ * @ff.info Please note that the default value of {@code trimSpaces} is {@literal true}
```

Comment:
Helemaal niet bij stil gestaan, maar zijn deze niet inheritable? of juist niet, kan ook handig zijn!

Reply:
Zojuist overlegd: deze zijn overerfbaar, maar niet automatisch. Je zult @inheritDoc op een subclass moeten zetten, dan worden deze daar ook getoond.